user-guide/Troubleshooting/Procedures/Investigating_OpenSearch_Issues.md

```diff
@@ -0,0 +1,165 @@
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) [?:?]
+  Caused by: org.opensearch.transport.RemoteTransportException: [DataNodeName][166.206.186.147:9300][internal:cluster/coordination/join/validate]
+  Caused by: org.opensearch.cluster.coordination.CoordinationStateRejectedException: join validation on cluster state with a different cluster uuid tNh7sXJeQjuf-RTTfFt7qg than local cluster uuid qVI2q9lMSy-Ot7O9v68d_A, rejecting
+          at org.opensearch.cluster.coordination.JoinHelper.lambda$new$4(JoinHelper.java:219) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.indexmanagement.rollup.interceptor.RollupInterceptor$interceptHandler$1.messageReceived(RollupInterceptor.kt:113) ~[?:?]
+          at org.opensearch.performanceanalyzer.transport.PerformanceAnalyzerTransportRequestHandler.messageReceived(PerformanceAnalyzerTransportRequestHandler.java:43) ~[?:?]
+          at org.opensearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:106) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.transport.InboundHandler$RequestHandler.doRun(InboundHandler.java:453) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:806) ~[opensearch-2.8.0.jar:2.8.0]
+          at org.opensearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:52) ~[opensearch-2.8.0.jar:2.8.0]
+          at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
+          at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
+          at java.lang.Thread.run(Thread.java:833) ~[?:?]
+  ```
+
+To fix this issue:
+
+1. Stop OpenSearch on the cluster manager and data nodes.
+
+1. Check the *cluster.initial_master_nodes* and *discovery.seed_hosts* on all nodes for any differences.
+
+1. Go to the data folder specified in the *opensearch.yml* file and delete the folder called *nodes* along with every file in it by using the following command:
+
+   ```bash
+   sudo rm -rf nodes
+   ```
+
+1. Restart OpenSearch on the cluster manager first and then on the data nodes.
+
+> [!IMPORTANT]
+> Deleting the nodes folder will result in loss of data. You should only do so with a new installation of OpenSearch.
+
+## SLSearch.txt logging mentions OpenSearch version is not officially supported
+
+If the *SLSearch.txt* log file mentions that the OpenSearch version is not officially supported, you can resolve this by upgrading your DMS to DataMiner 10.3.6 or higher.
+
+However, note that this has no functional impact, as the DMA will run fine even if you have not upgraded yet.
+
+## OpenSearch service going into timeout
+
```

Comment:
Maybe a bit more context? This happens when starting the service on the Linux machine and is one of I assume several issues that may occur when attempting to start the service. 
I haven't ran into this yet, but it might be interesting to know why it is timing out, that might be beyond the scope of this article though.

Reply:
Not sure why suddenly one node stopped working, but we needed to get it up and running. Quick googling showed ElasticSearch sometimes showcased the same problem.